v1/examples/webkit/.sauce/config.yml

```diff
@@ -11,15 +11,15 @@ sauce:
       - cypress-example
 
 cypress:
-  version: 12.3.0
+  version: 12.6.0
   configFile: "cypress.config.js"
 
 rootDir: ./
 
 suites:
   - name: (Experimental) webkit test
     browser: webkit
-    platformName: Windows 11
+    platformName: macOS 12
```

Comment:
Is this change intentional? And if so, maybe we should test against both platforms here?

Reply:
Yes, to make the example work.
It appears that on macOS 11 webkit is crashing at startup, on Windows 11, it's timeouting (Cypress loads, but never goes after the `Loading your tests` step.
It's working fine on macOS 12. I moved to this to have working example while I'm troubleshooting this.

Also, I'll add a bigger scope regarding runner testing, to avoid surprises like this.